HLTrigger/JetMET/plugins/HLTScoutingPFProducer.cc

```diff
@@ -220,29 +263,30 @@ void HLTScoutingPFProducer::produce(edm::StreamID sid, edm::Event &iEvent, edm::
           }
         }
       }
-      outPFJets->emplace_back(jet.pt(),
-                              jet.eta(),
-                              jet.phi(),
-                              jet.mass(),
-                              jet.jetArea(),
-                              jet.chargedHadronEnergy(),
-                              jet.neutralHadronEnergy(),
-                              jet.photonEnergy(),
-                              jet.electronEnergy(),
-                              jet.muonEnergy(),
-                              jet.HFHadronEnergy(),
-                              jet.HFEMEnergy(),
-                              jet.chargedHadronMultiplicity(),
-                              jet.neutralHadronMultiplicity(),
-                              jet.photonMultiplicity(),
-                              jet.electronMultiplicity(),
-                              jet.muonMultiplicity(),
-                              jet.HFHadronMultiplicity(),
-                              jet.HFEMMultiplicity(),
-                              jet.hoEnergy(),
-                              tagValue,
-                              0.0,
-                              candIndices);
+      outPFJets->emplace_back(
+          MiniFloatConverter::reduceMantissaToNbitsRounding(jet.pt(), mantissaPrecision_),
+          MiniFloatConverter::reduceMantissaToNbitsRounding(jet.eta(), mantissaPrecision_),
+          MiniFloatConverter::reduceMantissaToNbitsRounding(jet.phi(), mantissaPrecision_),
+          MiniFloatConverter::reduceMantissaToNbitsRounding(jet.mass(), mantissaPrecision_),
+          MiniFloatConverter::reduceMantissaToNbitsRounding(jet.jetArea(), mantissaPrecision_),
+          MiniFloatConverter::reduceMantissaToNbitsRounding(jet.chargedHadronEnergy(), mantissaPrecision_),
+          MiniFloatConverter::reduceMantissaToNbitsRounding(jet.neutralHadronEnergy(), mantissaPrecision_),
+          MiniFloatConverter::reduceMantissaToNbitsRounding(jet.photonEnergy(), mantissaPrecision_),
+          MiniFloatConverter::reduceMantissaToNbitsRounding(jet.electronEnergy(), mantissaPrecision_),
+          MiniFloatConverter::reduceMantissaToNbitsRounding(jet.muonEnergy(), mantissaPrecision_),
+          MiniFloatConverter::reduceMantissaToNbitsRounding(jet.HFHadronEnergy(), mantissaPrecision_),
+          MiniFloatConverter::reduceMantissaToNbitsRounding(jet.HFEMEnergy(), mantissaPrecision_),
+          jet.chargedHadronMultiplicity(),
+          jet.neutralHadronMultiplicity(),
+          jet.photonMultiplicity(),
+          jet.electronMultiplicity(),
+          jet.muonMultiplicity(),
+          jet.HFHadronMultiplicity(),
+          jet.HFEMMultiplicity(),
+          MiniFloatConverter::reduceMantissaToNbitsRounding(jet.hoEnergy(), mantissaPrecision_),
+          tagValue,
```

Comment:
Question (only noticing now): is `tagValue` supposed to maintain full precision?

Reply:
The reduced precision of the jets was added by @dsperka, so I better check with him.

Reply:
In the current configuration the tagValue is not filled, it is always at its default value of -20. I think we could put the reduced precision here, it will not make any difference in the current configuration and if we eventually fill the variable with something meaningful then the reduced precision should be fine.

Reply:
same goes for the "0.0" afterwards

Reply:
OK! I updated both `tagValue` and the 0.0 afterwards to reduced precision.